jaas/src/main/java/com/griddynamics/jagger/jaas/service/DynamicDataService.java

```diff
@@ -38,22 +37,25 @@
     
     private static final Logger LOGGER = LoggerFactory.getLogger(DynamicDataService.class);
     
-    private final ExecutorService destroyerService =
-            Executors.newSingleThreadExecutor(r -> new Thread("DynamicDataServiceDestoyer"));
+    private final ExecutorService destroyerService = Executors.newSingleThreadExecutor(r -> {
```

Comment:
I consider we should stick to KISS principle. So, (r -> new Thread(Runnable r, String name)) seems to be less complicated.